src/ert/run_models/base_run_model.py

```diff
@@ -413,6 +413,12 @@ def _get_number_of_finished_realizations_from_reruns(self) -> int:
             False
         ) - self._initial_realizations_mask.count(False)
 
+    def get_current_snapshot(self) -> EnsembleSnapshot:
+        if self._iter_snapshot.keys():
+            current_iter = max(list(self._iter_snapshot.keys()))
+            return self._iter_snapshot[current_iter]
+        return EnsembleSnapshot()
```

Comment:
Is this needed? This is for cases where the run model has no snapshots? Maybe it should be an error?

Reply:
> Is this needed?

Not really, but makes handling calls easier. 

> This is for cases where the run model has no snapshots ?

No this is for cases when this is called before calling `run_experiment` on the run_model.

> Maybe it should be an error ?
Why? As far as I can tell there is no added benefit for throwing an error.